lib/transforms/acceptance/visit.js

```diff
@@ -0,0 +1,43 @@
+const { makeAwait, dropAndThen, addImportStatement } = require('../../utils');
+
+/**
+ * Check if `node` is a `visit(url)` expression
+ *
+ * @param j
+ * @param node
+ * @returns {*|boolean}
+ */
+function isGlobalHelperExpression(j, node) {
+  return j.CallExpression.check(node)
+    && j.Identifier.check(node.callee)
+    && node.callee.name === 'visit';
+}
+
+/**
+ * Transform `visit(url)` to `await visit(url)`, remove `andThen` calls
+ *
+ * @param file
+ * @param api
+ * @returns {*|string}
+ */
+function transform(file, api) {
+  let source = file.source;
+  let j = api.jscodeshift;
+
+  let root = j(source);
+
+  let replacements = root
+    .find(j.CallExpression)
+    .filter(({ node }) => isGlobalHelperExpression(j, node))
+    ;
+
+  if (replacements.length > 0) {
+    makeAwait(j, replacements);
+    dropAndThen(j, root);
```

Comment:
I was thinking about "enqueuing" the `andThen` transform instead of running it several times. If that it how it works already ignore me.